tests/bdds/app-cleanup_test.go

```diff
@@ -0,0 +1,103 @@
+package appcleanup
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	scheme "k8s.io/client-go/kubernetes/scheme"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/litmuschaos/chaos-operator/pkg/apis/litmuschaos/v1alpha1"
+	chaosClient "github.com/litmuschaos/chaos-operator/pkg/client/clientset/versioned/typed/litmuschaos/v1alpha1"
+	restclient "k8s.io/client-go/rest"
+)
+
+var (
+	kubeconfig string
+	config     *restclient.Config
+	client     *kubernetes.Clientset
+	clientSet  *chaosClient.LitmuschaosV1alpha1Client
+)
+
+func TestChaos(t *testing.T) {
+
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "BDD test")
+}
+
+var _ = BeforeSuite(func() {
+
+	var err error
+	kubeconfig = os.Getenv("HOME") + "/.kube/config"
```

Comment:
Instead of this generate the clientSet using `-kubeconfig` flag, You can write a helper function whose task is to generate a clientset using flag

Reply:
In the next PR you can do this create an issue for the same

Reply:
Okay @chandankumar4  Thanks!